internal/security/bootstrapper/command/setupacl/command.go

```diff
@@ -167,17 +167,33 @@ func (c *cmd) Execute() (statusCode int, err error) {
 		return interfaces.StatusCodeExitWithError, fmt.Errorf("failed to configure Consul access: %v", err)
 	}
 
-	// write a sentinel file to indicate Consul ACL bootstrap is done so that we don't bootstrap ACL again,
-	// this is to avoid re-bootstrapping error and that error can cause the snap crash if restart this process
-	if err := c.writeSentinelFile(); err != nil {
-		return interfaces.StatusCodeExitWithError, fmt.Errorf("failed to write sentinel file: %v", err)
+	// create EdgeX service policy
```

Comment:
Right here is where you can validate the bootstrapACLToken.SecretID is set. edgeXServicePolicyName is a constant, so it doesn't need to be checked.

Reply:
I don't agree: so whoever calls that func has to check every time..this is too much